JavaScript tracing docs refactor (#12769)
getsentry/sentry-docs: docs/platforms/javascript/common/tracing/span-metrics/examples.mdx

---
title: Instrumentation Examples

> description: "Examples of using span metrics to debug performance issues and monitor application behavior across frontend and backend services."

Reviewer: ```suggestion
description: "How you can use span metrics to debug performance issues and monitor application behavior across frontend and backend services."
```
